data-raw/param.R

```diff
@@ -0,0 +1,12 @@
+library(data.table)
+
+param <- list(
+  DD_lt_0 = fread(file = "inputs/derivedVariables/optimizedParameterTables/param_DD_S1.csv"),
+  DD_gt_5 = fread(file = "inputs/derivedVariables/optimizedParameterTables/param_DD_S2.csv"),
+  DD_lt_18 = fread(file = "inputs/derivedVariables/optimizedParameterTables/param_DD_S3.csv"),
+  DD_gt_18 = fread(file = "inputs/derivedVariables/optimizedParameterTables/param_DD_S4.csv"),
+  NFFD = fread(file = "inputs/derivedVariables/optimizedParameterTables/param_NFFD.csv"),
+  PAS = fread(file = "inputs/derivedVariables/optimizedParameterTables/param_PAS.csv")
```

Comment:
Since they are dt, we could leverage `data.table::fread(key = , index = )` to index table.
They would probably stay R because they are not part of the dynamic bilinear interpolation stack.